chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/ProductManager.java

```diff
@@ -131,6 +141,9 @@ public ProductDTO updateProductOwnership(ProductDTO dto)
     @Transactional(readOnly = false)
     @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).PRODUCT, "
             + "T(gov.healthit.chpl.permissions.domains.ProductDomainPermissions).UPDATE, #dto)")
+    @CacheEvict(value = {
+            CacheNames.COLLECTIONS_LISTINGS
```

Comment:
Should probably evict the Product Names cache: `CacheNames.PRODUCT_NAMES`

Reply:
Done.